spec/features/idv/get_proofing_results_job_scenarios_spec.rb

```diff
@@ -578,9 +578,9 @@
 
       # Then the user is taken to the /verify/welcome page
       expect(current_path).to eq(idv_welcome_path)
-      # And the user has an InPersonEnrollment with status "passed"
+      # And the user has an InPersonEnrollment with status "cancelled"
```

Comment:
I can see cancelled and canceled in this file (status, comments, analytics).  If possible not too much trouble can you make it be consistent. If you think some of the changes are breaking maybe we should create a new ticket to handle this.

Reply:
This is seemingly an app-wide problem, so I'd +1 the new-ticket approach.

~Although if someone cancels that ticket, how will the Jira ticket status read?~

Reply:
<details>
<summary>Jira Has Spoken</summary>

![JiraHasSpoken](https://github.com/user-attachments/assets/b0e0316f-4d55-45ca-b9f1-891abf43dc19)

</details>